edb/pgsql/parser/ast_builder.py

```diff
@@ -800,6 +869,14 @@ def _build_a_expr(n: Node, c: Context) -> pgast.BaseExpr:
             expr=_build_base_expr(n["rexpr"], c),
             context=_build_context(n, c),
         )
+    elif n['kind'] == 'AEXPR_NULLIF':
```

Comment:
Why is postgres like this

Reply:
I'm sure there are quite a lot non-handled cases still waiting to be discovered, but I just don't have the motivation to go and do them. There also just too, too much of them:

https://github.com/pganalyze/libpg_query/blob/15-latest/protobuf/pg_query.proto

I'll do them when they are needed for something.